src/test/java/lotto/domain/BuyerTest.java

```diff
@@ -0,0 +1,86 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class BuyerTest {
+
+    @Test
+    void create() {
+        List<Lotto> lottoList = new ArrayList<>();
+        List<Integer> lottoNumber = Arrays.asList(1,2,3,4,5,6);
+
+        lottoList.add(new Lotto(lottoNumber));
+        Lottos lottos = new Lottos(lottoList);
+
+        Buyer buyer = new Buyer(lottos);
+
+        assertThat(buyer.getLottoNumbers().get(0)).isEqualTo(lottoNumber);
+        assertThat(buyer.getBuyCount()).isEqualTo(1);
+
+    }
+
+
+
+    @Test
+    void create_usingNumberGenInterface() {
+
+        List<Integer> lottoNumber = Arrays.asList(1,2,3,4,5,6);
+
+        Buyer buyer = new Buyer(1, new TestLottoNumberGenerator());
+        assertThat(buyer.getLottoNumbers().get(0)).isEqualTo(lottoNumber);
+        assertThat(buyer.getBuyCount()).isEqualTo(1);
+        assertThat(buyer.getWinningResult(lottoNumber).get(LottoRankingEnum.FIRST_PRIZE)).isEqualTo(1);
+    }
+
+    public static class TestLottoNumberGenerator implements LottoNumberGenerator {
+
+        @Override
+        public List<List<Integer>> genLottoNumbers(int cnt) {
+            return Arrays.asList(Arrays.asList(1,2,3,4,5,6));
+        }
+    }
```

Comment:
```suggestion
    public static class TestLottoNumberGenerator implements LottoNumberGenerator {

        private final List<Integer> values;

        public TestLottoNumberGenerator(Integer... values) {
            this.values = Arrays.asList(values);
        }

        @Override
        public List<List<Integer>> genLottoNumbers(int cnt) {
            return Arrays.asList(values);
        }
    }
```

이렇게 입력받게 구현하는건 어떨까요?

`Buyer buyer = new Buyer(1, new TestLottoNumberGenerator(1,2,3,4,5,6))`

그럼 다른 사람이 테스트코드를 볼때 의미가 잘 전달될것 같아요